python/py-imagehash/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-imagehash
+python.rootname     ImageHash
+version             4.3.1
+
+categories-append   devel graphics
+platforms           {darwin any}
+supported_archs     noarch
+license             custom
```

Comment:
why do you list this as `custom`, that's not a valid license and according to [upstream](https://github.com/JohannesBuchner/imagehash/blob/v4.3.1/LICENSE) it's `BSD2`, so shouldn't is just be `BSD`

Reply:
Thanks! Done.